packages/mask/src/extension/popups/pages/Friends/index.tsx

```diff
@@ -25,10 +26,12 @@ const Contacts = memo(function Contacts() {
         <Suspense fallback={<LoadingPlaceholder />}>
             {matchDetail ? null : <NormalHeader />}
             <RestorableScrollContext.Provider>
-                <Routes>
-                    <Route path="*" element={<Home />} />
-                    <Route path={`${r(PopupRoutes.FriendsDetail)}/:id?`} element={<FriendsDetail />} />
-                </Routes>
+                <QueryClientProvider client={new QueryClient()}>
```

Comment:
Why creating another QueryClient?

Reply:
Need a query client context otherwise can't invalidate the query